lazyflow/metaDict.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, other=None, *args, **kwargs):
             # are ready
             self._ready = False
 
+        if not 'has_mask' in self:
+            self.has_mask = False
+
```

Comment:
I did notice that keys/attributes that weren't present would return `None`. However, as the key/attribute name is `has_mask`, I thought it best to make it unambiguously boolean. This way the "end user" would not have any surprises by making the naive assumption that it is strictly boolean. As this was done with `_ready`, I did not think this was an issue. If you still think it should be changed, I would be happy to do so.